pom.xml

```diff
@@ -98,12 +113,14 @@
         <dependency>
             <groupId>com.vaadin</groupId>
             <artifactId>vaadin-client</artifactId>
-            <scope>provided</scope>
+            <version>${vaadin.version}</version>
+            <scope>compile</scope>
```

Comment:
why the scope change?

Reply:
ah my bad, needed it to compile it locally